src/Features/Navigation/Menu.php

```diff
@@ -399,6 +399,13 @@ public function add_core_items() {
 
 		$items = CoreMenu::get_items();
 		foreach ( $items as $item ) {
+			if ( ! isset( $item['is_top_level'] ) && ! isset( $item['parent'] ) ) {
+				if ( isset( $item['is_category'] ) && $item['is_category'] ) {
+					self::add_plugin_category( $item );
+				} else {
+					self::add_plugin_item( $item );
+				}
+			}
 			self::add_item( $item );
```

Comment:
Wouldn't this line result in the items being added twice (or attempting to), since `self::add_plugin_item()` calls `self::add_item()` as well?

Reply:
Good call.  You're right and this bug wasn't showing since we don't allow duplicate IDs.  Thanks for catching this.